example-webhook.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: poad-apiserver-env.networking.gardener.cloud
```

Comment:
`poad-apiserver-env.networking.gardener.cloud` replace by `pod-apiserver-env.networking.gardener.cloud`

Reply:
What about `apiserver-proxy.networking.gardener.cloud`? That is the name I've chosen for the gardener [implementation](https://github.com/gardener/gardener/pull/3082/files#diff-e6e1e3ff11852b16b125d715a186b087904794f53f251198edf6056d69e92ba2R4)